include/fluent-bit/flb_filter.h

```diff
@@ -87,6 +87,7 @@ struct flb_filter_instance {
 #ifdef FLB_HAVE_REGEX
     struct flb_regex *match_regex; /* match rule (regex) based on Tags */
 #endif
+    void *parent;                  /* Parent processor         */
```

Comment:
I suggest to rename this variable since `parent` is commonly used in other cases to refer a parent context that holds the current instance, maybe to `parent_processor` just to avoid confusions when reading the code (the definition is clear, but not in the code when using ->parent)